installer/helm/chart/volcano/templates/kubestatemetrics.yaml

```diff
@@ -1,4 +1,5 @@
 {{- if .Values.custom.metrics_enable }}
+{{ $kube_state_metrics_ns := or .Values.custom.admission_ns .Values.custom.kube_state_metrics_ns }}
```

Comment:
Why add `Values.custom.admission_ns` here?

Reply:
looks a bad copy/paste will remove that, also should I set kube_state_metrics_ns default value equals to current value ?